modules/openy_gc_auth/modules/openy_gc_auth_personify/src/Controller/PersonifyAuthController.php

```diff
@@ -135,8 +149,9 @@ public function auth(Request $request) {
           ]);
         }
         else {
-          user_cookie_delete('personify_authorized');
-          user_cookie_delete('personify_time');
+          $request->cookies->set('Drupal_visitor_personify_authorized', $token);
+          $event = new GCUserLogoutEvent();
+          $this->eventDispatcher->dispatch(GCUserLogoutEvent::EVENT_NAME, $event);
```

Comment:
In this case, you will execute all possible event subscribers that will invoke GCUserLogoutEvent::EVENT_NAME.
If any of the developers will add their custom code here, it will be executed as well.
It is better to move code from personify logout event subscriber to the trait or service and execute it from both places.